packages/app/src/server/routes/apiv3/revisions.js

```diff
@@ -128,7 +127,7 @@ module.exports = (crowi) => {
       const paginateResult = await Revision.paginate(
         { pageId: page._id },
         {
-          page: selectedPage,
+          offset,
           limit,
           sort: { createdAt: -1 },
           populate: 'author',
```

Comment:
Does this change affect this code?

https://github.com/weseek/growi/blob/b8117fc0e0344f88afe75fde08a2ef98c16fd1c0/packages/app/src/stores/page.tsx#L146-L164